avocado/core/html.py

```diff
@@ -130,20 +130,21 @@ def tests(self):
                    "INTERRUPTED": "danger"}
         test_info = []
         results_dir = self.results_dir(False)
-        for tst in self.result.tests:
-            tst = tst.copy()    # we don't want to override other's results
-            tst["test"] = str(tst["name"])
-            logdir = os.path.join(results_dir, 'test-results', tst['logdir'])
-            tst['logdir'] = os.path.relpath(logdir, self.html_output_dir)
+        for t in self.result.tests:
+            formatted = {}
+            formatted['name'] = t['name']
```

Comment:
it'd be safer to use `str` here, but I'm fine with this if it works fine (I'll check it in functional review later)